build.gradle

```diff
@@ -34,3 +34,57 @@ allprojects {
         ]
     }
 }
+
+// 0. clean up old files and register that task
+tasks.register('cleanUpChromeFolders', Delete) {
+    delete file('./chrome-binary'),
+            file('./chrome-driver')
+}
+tasks.named("clean") {
+    dependsOn('cleanUpChromeFolders')
+}
+
+// 1. create folders
+tasks.register('prepareDirs') {
+    mustRunAfter clean
+    doFirst {
+        mkdir "./chrome-binary/"
```

Comment:
1. Add download folder(s) to `.gitignore`
2. It's a nitpicking, but just use the same folder